### ISSUE TITLE
[BUG/ISSUE]  SPC_RESTART or GEOSCHEM_RESTARTS?

### ISSUE THREAD

**Issue by LiamBindle** [geoschem/GCHP#45]
## Describe the bug:

Is ExtData/SPC_RESTARTS or ExtData/GEOSCHEM_RESTARTS the proper place for our restarts? Currently, `./createRunDir.sh` links to restarts in SPC_RESTARTS, but those don't exist on ComputeCanada. I assume `./createRunDir.sh` needs to be updated--is that right?

**Comment by lizziel** [comment]
In the ExtData we have at Harvard there is a symbolic link SPC_RESTARTS within ExtData to allow backwards compatibility across versions. I thought ComputeCanada had a mirror image of ExtData at Harvard, but maybe not? Regardless, I checked the latest dev/13.0.0 createRunDir.sh and the symbolic link is set to GEOSCHEM_RESTARTS:
`restarts=${GC_DATA_ROOT}/GEOSCHEM_RESTARTS`

Are you using an older version? I updated the restart path a few weeks ago when I created new restarts for 13.0.0.
